kunstmaan/dashboard-bundle/5.2/config/packages/kunstmaan_dashboard.yaml

```diff
@@ -0,0 +1,9 @@
+# Configurate the dashboard bundle google api parameters to enable the google widget
+# More info at https://kunstmaanbundlescms.readthedocs.io/en/latest/cookbook/google-analytics-dashboard/
+#kunstmaan_dashboard:
+#    google_analytics:
+#        api:
+#            client_id: ''
+#            client_secret: ''
```

Comment:
You should use environment variables or remove this file

Reply:
@Devolicious Do we remove this files and extend our documention so it's clear what the user should do, or do we add env vars for the google analytics config? I'm more in favor of removing it

Reply:
@acrobat I would add the env vars. We use it all the time so it's more convenient if we do it that way. I know that those vars are 99% of the time not really bound to a specific environment but then the option will be there and if the user wants to replace with fixed vars that is also possible.